fe/fe-core/src/main/java/com/starrocks/analysis/DateLiteral.java

```diff
@@ -50,9 +50,9 @@
 public class DateLiteral extends LiteralExpr {
     private static final Logger LOG = LogManager.getLogger(DateLiteral.class);
 
-    private static final DateLiteral MIN_DATE = new DateLiteral(0, 1, 1);
+    private static final DateLiteral MIN_DATE = new DateLiteral(1970, 1, 1);
     private static final DateLiteral MAX_DATE = new DateLiteral(9999, 12, 31);
-    private static final DateLiteral MIN_DATETIME = new DateLiteral(0, 1, 1, 0, 0, 0);
+    private static final DateLiteral MIN_DATETIME = new DateLiteral(1970, 1, 1, 0, 0, 0);
```

Comment:
refer to  BE src/runtime/time_types.h codes, seems you shouldn't change here.
```
    // from_date(1970, 1, 1)
    static constexpr JulianDate UNIX_EPOCH_JULIAN = 2440588;
    // from_date(0000, 0, 0)
    static constexpr JulianDate ZERO_EPOCH_JULIAN = 1721028;
    // from_date(0000, 1, 1)
    static constexpr JulianDate BC_EPOCH_JULIAN = 1721060;
    // from_date(0001, 1, 1)
    static constexpr JulianDate AD_EPOCH_JULIAN = 1721426;

    // from_date(0000, 1, 1)
    static constexpr JulianDate MIN_DATE = 1721060;

    // from_date(9999, 12, 31)
    static constexpr JulianDate MAX_DATE = 5373484;
```

Reply:
+1，This will change the user's behavior, for example, the user's previous LESS than partition can receive all the time, so there may be some import failures.